src/components/AppControls/AppControls.vue

```diff
@@ -322,7 +347,7 @@ export default {
     z-index: 2;
 
     /* The height of the nextcloud header */
-    top: var(--header-height);
+    top: 0px;
```

Comment:
Maybe this can be completely removed at this point?

Reply:
Then, the element is no longer sticky.

Reply:
You're right. My bad